mvt/layer_test.go

```diff
@@ -129,31 +130,77 @@ func TestLayerAddFeatures(t *testing.T) {
 }
 
 func TestLayer(t *testing.T) {
+	tile := tegola.NewTile(0, 0, 0)
+
 	baseBBox := tegola.BoundingBox{
 		Minx: 0,
 		Miny: 0,
 		Maxx: 4096,
 		Maxy: 4096,
 	}
-	testcases := []struct {
+	fromPixel := func(x, y float64) *basic.Point {
```

Comment:
method is being repeated in TestEncodeGeometry. Maybe pull out of individual tests and put in mvt_test.go file?

Reply:
It is being used to help build out the test cases, instead of doing the math manually to convert from pixel coords to webmarcator for the lines, and polygons of a feature.